docker/django/docker-entrypoint.sh

```diff
@@ -20,12 +20,20 @@ if [[ "$CREATE_SUPERUSER" = "true" ]]; then
     ./manage.py create_admin_superuser
 fi
 
+if [[ "$COLLECT_STATIC" = "true" ]]; then
```

Comment:
Another common solution for doing this would be to have a production docker image (additional build stage in the `Dockerfile`) where templates would be installed as part of the build process.

Reply:
Exmples:
- https://github.com/City-of-Helsinki/maritime-maas/blob/main/Dockerfile
- https://github.com/City-of-Helsinki/kukkuu/blob/master/Dockerfile

Reply:
Running `collectstatic` in the `Dockerfile` will make the static files part of the image. Doing it in the entrypoint will of course allow the image be more dynamic, but it will add more to the startup time for the container. Not much, but some still.

Is there a situation where we would not like to run `collectstatic` in a production environment? In your local dev environment (development image), at least usually, you don't want/need `collectstatic` to happen.

`install_templates` looks like something we might not want to have in every situation, so maybe having at least that in the entrypoint script is good.

Reply:
That said, the current solution also works and we can refactor it later if needed, your call. :+1: :shipit:

Reply:
I would remove the COLLECT_STATIC variable and bind static collecting to the DEBUG variable. Needing to run static collecting when debug is true should be very rare, you can still do it pretty easily manually. Having fewer settings and more consistency between the projects in Platta will be beneficial in the future at least for our team 🙂